tests/sentry/issues/endpoints/test_organization_issue_metrics.py

```diff
@@ -209,8 +238,8 @@ def test_get_feedback(self) -> None:
                     "valueUnit": None,
                 },
                 "values": [
-                    {"timestamp": int(prev.timestamp()), "value": 1},
-                    {"timestamp": int(curr.timestamp()), "value": 1},
+                    {"timestamp": int(prev.timestamp()), "value": 2},
+                    {"timestamp": int(curr.timestamp()), "value": 2},
```

Comment:
this test was just wrong -- there were 4 created Groups -- the two resolved ones had ~sorta undefined beginning dates so they didn't get included in the range

after fixing the dates I adjusted the values here